packaging/windows/amazon-cloudwatch-agent-ctl.ps1

```diff
@@ -449,15 +449,13 @@ Function CWOCConfig() {
         Copy-Item "${PREDEFINED_CONFIG_DATA}" -Destination "${YAML_DIR}/default.tmp"
         Write-Output "Successfully fetched the config and saved in ${YAML_DIR}\default.tmp"
     } else {
-        & $CWAProgramFiles\config-downloader.exe --output-dir "${YAML_DIR}" --download-source "${OtelConfigLocation}" --mode "${param_mode}" --config "${COMMON_CONIG}" --multi-config "${multi_config}"
-        if ($LASTEXITCODE -ne 0) {
-            return
-        }
+        & cmd /c "`"$CWAProgramFiles\config-downloader.exe`" --output-dir ${YAML_DIR} --download-source ${OtelConfigLocation} --mode ${param_mode} --config ${COMMON_CONIG} --multi-config ${multi_config} 2>&1"

```

Comment:
In the PR overview, change description: it says "using cmd/ c for config-translator". I think you meant to say "using cmd/ c for config-downloader.exe". Nice work figuring out this change fix the issue, before we merge this change, can you do a quick spike regarding understand, why `& $CWAProgramFiles\config-downloader.exe ...` is used in the first place, while `amazon-cloudwatch-agent` and `config-translator` is invoked by using cmd /c? I don't have the context the different between those two, and I think understand this difference better could help us avoid side effects this change may cause.

Reply:
> In the PR overview, change description: it says "using cmd/ c for config-translator". I think you meant to say "using cmd/ c for config-downloader.exe".
Ack. My bad on writing the wrong artifacts. 

> Nice work figuring out this change fix the issue, before we merge this change, can you do a quick spike regarding understand, why & $CWAProgramFiles\config-downloader.exe ... is used in the first place, while amazon-cloudwatch-agent and config-translator is invoked by using cmd /c
That's a good question. Besides the advantage/disadvatange between using `cmd /c` and native shell, I have not know why it was used in the first place.